tests/conftest.py

```diff
@@ -45,3 +152,47 @@ def mocked_docker_client():
     docker_client.images.build.side_effect = docker_images
     docker_client.images.list.return_value = [docker_image]
     return docker_client
+
+
+@pytest.fixture(
+    params=[
+        "transmogrifier-example-job-1-abc123:latest",
+        "transmogrifier-example-job-1-def456:latest",
+    ]
+)
+def mocked_docker_container_and_image(
+    request, mocked_docker_container_a, mocked_docker_container_b
+):
+    if "abc123" in request.param:
+        yield mocked_docker_container_a, request.param
+    elif "def456" in request.param:
+        yield mocked_docker_container_b, request.param
```

Comment:
See docstring in the test for [`run_docker_container_success`](https://github.com/MITLibraries/transmogrifier-ab-diff/pull/6/files#diff-ef381e309d69d9280a44bcd8e1f647b38878fd2167b92011767f6f351ce4363dR52-R89).